browser/src/control/Control.JSDialog.js

```diff
@@ -501,6 +501,13 @@ L.Control.JSDialog = L.Control.extend({
 		var offsetX = isSpreadsheetRTL ? 0 : app.sectionContainer.getSectionWithName(L.CSections.RowHeader.name).size[0];
 		var offsetY = app.sectionContainer.getSectionWithName(L.CSections.ColumnHeader.name).size[1];
 
+
+		var isDescendant = instance.lastFocusedElement.closest('table#menu.jsdialog.autofilter.ui-treeview');
```

Comment:
doesn't that work only when focus in some specific place?

Reply:
Yes, this will be our key point that the current opened popup will surly the child autofilter popup.

Reply:
and also it will be obvious that the current popup is child popup only and only is the last focused this is at parent .

so i think it will be a good way for calculate child popup.

@eszkadev is there any possibility or way that the last focused thing will not be parent autofilter popup in case of opening a child autofilter popup ? ( i think it will not possible )

Reply:
I think it might be not reliable. (we can always break this in the future).
Maybe try to guess that by looking at JSON structure?
Which widgets are visible, which not for example.